_chapters/chapter_01.md

```diff
@@ -238,11 +261,14 @@ layout: rails_tutorial
 {% aside %}
   What's this bookstore we keep talking about? In this tutorial, we're going to use Rails to make an application for a bookstore.
 
-  As you build the application, you'll add features to create, view, edit, and delete books. Since we all have feelings on the books we love...and hate, you'll add features so books can be reviewed by users.
+  As you build the application, you'll add features to `create`, `view`, `edit`, and `delete` books. Since we all have feelings on the books we love...and hate, you'll add features so books can be reviewed by users.
```

Comment:
I think create, view, etc. should be treated as words, not code. I was trying to translate the coding stuff into people talk.